certified-connectors/Emigo/apiProperties.json

```diff
@@ -0,0 +1,51 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "token": {
+        "type": "oauthSetting",
+        "oAuthSettings": {
+          "identityProvider": "aad",
+          "clientId": "<<Enter your client ID>>",
+          "scopes": [],
+          "redirectMode": "Global",
+          "redirectUrl": "https://global.consent.azure-apim.net/redirect",
+          "properties": {
+            "IsFirstParty": "False",
+            "AzureActiveDirectoryResourceId": "https://emigoapps.onmicrosoft.com/PowerPlatformConnectorEmigo",
+            "IsOnbehalfofLoginSupported": true
+          },
+          "customParameters": {
+            "loginUri": {
+              "value": "https://login.windows.net"
+            },
+            "tenantId": {
+              "value": "common"
+            },
+            "resourceUri": {
+              "value": "https://emigoapps.onmicrosoft.com/PowerPlatformConnectorEmigo"
+            },
+            "enableOnbehalfOfLogin": {
+              "value": "false"
+            }
+          }
+        }
+      },
+      "token:TenantId": {
+        "type": "string",
+        "metadata": {
+          "sourceType": "AzureActiveDirectoryTenant"
+        },
+        "uiDefinition": {
+          "constraints": {
+            "required": "false",
+            "hidden": "true"
+          }
+        }
+      }
+    },
+    "iconBrandColor": "#fbfbfb",
+    "capabilities": [],
+    "publisher": "Tomasz Margacz",
```

Comment:
@tomaszsagra for all our verified publishers we ask that the "publisher" and "stackOwner" properties be the same. Please change to "Sagra Technology"